src/pastel/EnableWinConsoleColors.re

```diff
@@ -0,0 +1,8 @@
+external enableWinConsoleAniSeq: unit => unit = "enable_windows_console_ansi_seqences";
```

Comment:
I think you meant for this to say `enableWinConsoleAnsiSeq` not `enableWinConsoleAniSeq`

Reply:
Fixed